### PR TITLE
Fix two errors found with AFL

### PR DESCRIPTION
**Pull request by Djack1010**
The project was fuzzed using AFL with ASAN. We wrote a testing program to make the library parse the input file provided by AFL. We analyzed the crashes and found two errors. We fixed them, checked for regression and then a new AFL run reported no crashes.

### CODE REVIEW

File: src/nanosvg.h [hunk 1]
```diff
@@ -279,14 +279,17 @@ static void nsvg__parseElement(char* s,
 
 	// Get attribs
 	while (!end && *s && nattr < NSVG_XML_MAX_ATTRIBS-3) {
+		char* attr_ptr = NULL;
```
Comment (memononen) [inline src/nanosvg.h]: let's call these `name` and `value`. That `_ptr` is not inline with the rest of the code naming style.

File: src/nanosvg.h [hunk 2]
```diff
@@ -279,17 +279,17 @@ static void nsvg__parseElement(char* s,
 
 	// Get attribs
 	while (!end && *s && nattr < NSVG_XML_MAX_ATTRIBS-3) {
-		char* attr_ptr = NULL;
-		char* value_ptr = NULL;
+		char* name = NULL;
+		char* value = NULL;
 
-		// Skip white space before the attrib name
+		// Skip white space before the attrib char* name = NULL;
```
Comment (memononen) [inline src/nanosvg.h]: Comment looks odd :)

**Comment by Djack1010** [comment]
Is there anything else that I need to change?

**Comment by memononen** [comment]
Nope, this looks good now. Me being slow :) Thanks for the fix!

**Pull request merged**